docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,713 @@
+If the goal of the analysis is to find potential differences in diffusion constants between different tracks, the following time-averaged estimator can be used:
+
+.. math::
+
+    \hat{\rho}[n] = \frac{1}{N - n} \sum_{i=1}^{N-n}\left(x_{i+n} - x_{i}\right)^2
+
+Here :math:`\hat{\rho}[n]` corresponds to the estimate of the MSD for lag :math:`n`, :math:`N` is the number of time points in the track, and :math:`x_i` is the track position at time frame :math:`i`.
+This particular estimator is referred to as the Time Averaged MSD (TAMSD), because the estimates for each lag are averaged over all time increments.
+Consider the definition of the TAMSD, it can be seen that the displacements are not independent from one another.
+While this definition results in a well-averaged estimate, this reduced uncertainty comes at a cost.
+Estimates obtained using this estimator are highly correlated :cite:`qian1991single,michalet2010mean,michalet2012optimal` which needs to be considered in subsequent analyses (more on this later).
+
+With Pylake, we can calculate the MSD for a :class:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack` with :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack.msd`.
+This returns a tuple of time and MSD values, which we can directly plot::
+
+    for track in tracks:
+        plt.plot(*track.msd())
+
+.. image:: figures/track_msd.png
+
+Note how the curves diverge quickly for larger time lags.
+Larger lags have far fewer points contributing to them.
+MSDs follow a gamma distribution becoming more and more Gaussian with more averaging :cite:`michalet2010mean,michalet2012optimal`.
+This means that smaller lags generally show a symmetric almost Gaussian distribution, while large (less averaged) lags show a much bigger variance.
+By simulating a large number of tracks, we can visualize this::
+
+    tracks = lk.simulation.simulate_diffusive_tracks(diffusion_constant=1.0, steps=12, dt=0.01, observation_noise=0.1, num_tracks=1500)
+    frame_lags = tracks[0].msd()[0]
+    all_msds = np.asarray([track.msd()[1] for track in tracks]).T
+    histogram_edges = np.arange(0, np.max(all_msds), 0.02)
+    msd_histogram = np.array([np.histogram(msd_frame, histogram_edges, density=True)[0] for msd_frame in all_msds])
+    histogram_centers = (histogram_edges[:-1] + histogram_edges[1:]) / 2
+
+    fig = plt.figure(figsize=(10, 5))
+    ax = fig.add_subplot(121, projection='3d')
+    view_limit = 0.8
+    for lag_time, hist in zip(frame_lags, msd_histogram):
+        ax.bar(
+            histogram_centers[histogram_centers < view_limit],
+            hist[histogram_centers < view_limit],
+            zs=lag_time,
+            zdir='y',
+            alpha=0.8,
+            width=histogram_centers[1]-histogram_centers[0]
+        )
+
+    plt.xlim([0, 0.8])
+    ax.view_init(20, -60, 0)
```

Comment:
I got an error here, and according to[ this ](https://matplotlib.org/stable/api/toolkits/mplot3d/view_angles.html) page, matplotlib does not have the 'roll' option for `a.view.init()`. Maybe just remove the zero?

Reply:
Good catch! Removing. The default is zero anyhow.